app/views/alaveteli_pro/plans/index.html.erb

```diff
@@ -32,6 +32,71 @@
   <p>We’re currently beta-testing these features. To be part of this closed test, please <a href="mailto:alaveteli-pro@mysociety.org">get in touch</a>.</p>
 </div>
 
+<div class="marketing__section">
+  <div class="marketing__section__heading">
+    <h2>WhatDoTheyKnow<sup>Pro</sup></h2>
+    <p>An all-in-one FOI toolkit for journalists, activists and campaigners</p>
+  </div>
+  <div class="marketing__section__content">
+    <div class="marketing__section__content-items">
+      <ul class="marketing__section__feature-list">
+        <li>Unlimited <strong class="marketing-highlight">private requests</strong>: break the story when you’re ready</li>
```

Comment:
Are we still happy with "Unlimited private requests" as we tweaked that for the leaflet IIRC